evap/staff/forms.py

```diff
@@ -333,6 +336,10 @@ def __init__(self, *args, **kwargs):
         self.fields['courses_participating_in'].initial = courses_of_current_semester.filter(participants=self.instance) if self.instance.pk else ()
         self.fields['courses_participating_in'].label = _("Courses participating in (active semester)")
         self.fields['courses_participating_in'].required = False
+        self.fields['is_staff'].initial = Group.objects.filter(user=self.instance, name="Staff").exists()
+        self.fields['is_staff'].label = _("Staff user")
+        self.fields['is_grade_user'].initial = Group.objects.filter(user=self.instance, name="Grade publisher").exists()
+        self.fields['is_grade_user'].label = _("Grade user")
```

Comment:
there are `self.instance.is_staff` and `self.instance.is_grade_publisher`. also, could you specify the label as parameter to `BooleanField`?